phoenix-core/src/it/java/org/apache/phoenix/end2end/HashJoinMoreIT.java

```diff
@@ -43,7 +43,7 @@
 
 import com.google.common.collect.Maps;
 
-public class HashJoinMoreIT extends BaseHBaseManagedTimeIT {
+public class HashJoinMoreIT extends BaseHBaseManagedTimeTableReuseIT {
```

Comment:
It doesn't seem like this test was converted to use random table names. Please revert the changes to this file.